dataflow/src/org/checkerframework/dataflow/util/PurityChecker.java

```diff
@@ -95,12 +96,16 @@ public static PurityResult checkPurity(
         protected final List<Pair<Tree, String>> notSeFreeReasons;
         protected final List<Pair<Tree, String>> notDetReasons;
```

Comment:
I assume that you'll remove this field when you remove the corresponding methods.